docker/transformers-all-latest-gpu/Dockerfile

```diff
@@ -3,6 +3,14 @@ LABEL maintainer="Hugging Face"
 
 ARG DEBIAN_FRONTEND=noninteractive
 
+# So far mainly used to specify the versions explicitly here, and not meant to be used as arguments for docker build.
+ARG PYTORCH='1.11.0'
+ARG TORCH_VISION='0.12.0'
```

Comment:
For `torch-scatter` and `intel_extension_for_pytorch`, can we **safely** assume that we can use `$PYTORCH`?
Otherwise, we can add `ARG TORCH_SCATTER_PT_VER=...` and `INTEL_TORCH_EXT_PT_VER = ...`.